parser/func.c

```diff
@@ -69,7 +70,7 @@ void func_def_debug(int depth, parser_node *node)
 {
     node_func_def *func = (node_func_def *)node->data;
     printtabs(depth);
-    if (func->statements)
+    if (func->statements->total)
```

Comment:
What if we have a actual function definition with empty body? ```void f() {}```. This will behave equally with `void f();`  (Not sure if it's important at all tho haha)